nemo/collections/nlp/modules/common/common_utils.py

```diff
@@ -50,7 +53,9 @@ def get_pretrained_lm_model(
         Pretrained model (NM)
     '''
     if pretrained_model_name in get_huggingface_lm_models_list():
-        model = get_huggingface_lm_model(config_file=config_file, pretrained_model_name=pretrained_model_name)
+        model = get_huggingface_lm_model(
+            config_dict=config_dict, config_file=config_file, pretrained_model_name=pretrained_model_name
+        )
```

Comment:
why?

Reply:
This is so we can instantiate BERT from yaml rather than having to read a .json.

Reply:
I don't think it's useful enough to add this

Reply:
It's really awkward to be configuring everything from yaml, and then ask the user to supply an additional .json file. This is a very simple change and keeps the .json functionality.